rules/CodingStyle/ClassNameImport/ShortNameResolver.php

```diff
@@ -171,7 +171,7 @@ private function resolveFromStmtsDocBlocks(array $stmts): array
             $phpDocNodeTraverser->traverseWithCallable(
                 $phpDocInfo->getPhpDocNode(),
                 '',
-                static function ($node) use (&$shortNames) {
+                static function (mixed $node) use (&$shortNames) {
```

Comment:
@TomasVotruba it seems it should be skipped as used as `Arg`, and the signature originally as callable `callable(Node $node)`:

```php
    /**
     * @param callable(Node $node): (int|null|Node) $callable
     */
    public function traverseWithCallable(Node $node, string $docContent, callable $callable): void
```